README.md

```diff
@@ -565,6 +566,62 @@ streams:
 
 Tested: KD110, KC200, KC401, KC420WS, EC71.
 
+#### Source: Tuya
+
+[Tuya](https://www.tuya.com/) proprietary camera protocol with **two way audio** support. Go2rtc supports `Cloud API` and `Open API`.
+
+The `Cloud API` requires setting up a cloud project in the Tuya Developer Platform to retrieve the required credentials. The `Open API` does not require a cloud project and the cameras can be added through the interface via QR code (user code required), but it does not support webrtc mode and two way audio.
+
+**Cloud API**:
+- Obtain `device_id`, `client_id`, `client_secret`, and `uid` (if using `mode=webrtc`) from [Tuya IoT Platform](https://iot.tuya.com/). [Here's a guide](https://xzetsubou.github.io/hass-localtuya/cloud_api/).
+
+**Open API**:
+- To get your user code, open the Tuya Smart app or Smart Life app and go to `Profile` > `Settings` > `Account and Security` > `User Code`
+- Open the Go2rtc interface and go to `Add` > `Tuya` and enter your `User Code` in the `User Code` field. Click on `Generate QR Code` and scan it with the Tuya Smart app or Smart Life app. After scanning, click on `Login`. All cameras in your home (not shared ones) will be listed in the Go2rtc interface. Copy/Paste stream URLs to your `go2rtc.yaml` file.
+
+**Configuring the stream:**
+- Use `mode` parameter to select the stream type (not all cameras support all modes):
+  - `webrtc` - WebRTC stream _(default for `Cloud API`)_
+  - `rtsp` - RTSP stream _(default for `Open API`)_
+  - `hls` - HLS stream
+  - `flv` - FLV stream _(only available for `Open API`)_
+  - `rtmp` - RTMP stream _(only available for `Open API`)_
+
+- Use `resolution` parameter to select the stream (only available for `Cloud API` and not all cameras support `hd` stream):
```

Comment:
I can visualize people opening issues that their cameras support HD/SD but this is not working, I think we can make it clearer:

```suggestion
- Use `resolution` parameter to select the stream (only available for `Cloud API` and not all cameras support `hd` stream through WebRTC even if supported in the camera itself):
```